prisma/schema.prisma

```diff
@@ -32,6 +33,17 @@ model ProfessorInfo {
   teachingReleaseExplanation String?
 }
 
+model ProfessorScore {
+  id                         Int              @id @default(autoincrement())
+  user                       User             @relation(fields: [userId], references: [id], onDelete: Cascade)
+  userId                     Int              @unique
+  comment                    String
+  teachingScore              Float
+  researchScore              Float
+  serviceScore               Float
+  totalScore                 Float
```

Comment:
is totalScore the same as the displayed calculated score? If so I think it would be better if we called it calculatedScore or something.

Reply:
if I understood Anna correctly before, then the total score isn't actually calculated from the other numbers, but rather is a separate score that the merit committee member picks. so i think `totalScore` might make more sense than `calculatedScore`, but lmk what you think

Reply:
ah gotchas, weirdge method from the mc's but in that case I think it's g then